README.md

```diff
@@ -163,6 +180,10 @@ When Reshuffle is running on a server, you can use the `authenticator` utility
 provided with this connector to start the authentication process from your
 (or your user's) machine.
 
+This action throws en error if the connector already has Salesforce access
```

Comment:
an error

Reply:
Fixed (this is eld English I wes oozing)